src/libraries/System.Text.RegularExpressions/gen/Resources/xlf/Strings.cs.xlf

```diff
@@ -192,9 +192,9 @@
         <target state="translated">Vypršel časový limit modulu Regex při pokusu o porovnání vzoru se vstupním řetězcem. K tomu může dojít z celé řady důvodů, mezi které patří velká velikost vstupních dat nebo nadměrné zpětné navracení způsobené vloženými kvantifikátory, zpětnými odkazy a dalšími faktory.</target>
         <note />
       </trans-unit>
-      <trans-unit id="RegexMemberMustHaveValidSignatureMessage">
-        <source>GeneratedRegexAttribute method or property must be partial, parameterless, non-generic, non-abstract, and return Regex. If a property, it must also be get-only.</source>
-        <target state="new">GeneratedRegexAttribute method or property must be partial, parameterless, non-generic, non-abstract, and return Regex. If a property, it must also be get-only.</target>
+      <trans-unit id="RegexMethodMustHaveValidSignatureMessage">
```

Comment:
This also seems to be undoing a change from June.  https://github.com/dotnet/runtime/commit/2bf492b0e5b95fa6914608a48658483034683fda#diff-4b415858dc6fa5baaeb017f57860b16e6f0e2479dd6092133aaeeb7765eb9c7bR195

Reply:
this is expected as far as I understand the loc system since the id changed.